insights/terraform/modules/jobs/policies.tf

```diff
@@ -0,0 +1,94 @@
+## Glue Role
+
+resource "aws_iam_role" "glue_role" {
+  name        = "bfd-insights-${var.project}-glue-role"
+  description = "Allow the Glue service to access the Insights buckets" 
+  tags        = var.tags
+  path        = "/bfd-insights/"
+
+  assume_role_policy = <<-POLICY
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Action": "sts:AssumeRole",
+        "Principal": {
+          "Service": "glue.amazonaws.com"
+        },
+        "Effect": "Allow",
+        "Sid": ""
+      }
+    ]
+  }
+  POLICY
+}
+
+## Find and create policies for the role
+
+data "aws_iam_policy" "glue_service" {
+  arn  = "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
+}
+
+data "aws_iam_policy" "athena_service" {
+  arn  = "arn:aws:iam::aws:policy/AmazonAthenaFullAccess"
+}
+
+data "aws_iam_policy_document" "s3_access" {
```

Comment:
Interesting way to generate policy documents!

Reply:
Right. JSON and HEREDOC are harder to debug.